includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -232,6 +242,7 @@ protected function re_add_meta_tags_in_optimized_order() {
 		$previous_meta_tag = null;
 		foreach ( $this->meta_tags as $meta_tag_group ) {
 			foreach ( $meta_tag_group as $meta_tag ) {
+				$meta_tag = $this->dom->importNode( $meta_tag );
```

Comment:
It should be in the document, however I had a weird test failure in one of the plugin tests.

It could be a problem of the test setup, though...

Do you think I should remove and we'll reinvestigate the cause of the test failure instead?

Reply:
Yeah, let's investigate the failure as it may be exposing some more fundamental problem.

Reply:
Alright, I'll remove the import again and try to debug the root cause.

Reply:
As it turns out, there's no test failures anymore without the import.

I made a few changes to avoid corrupting behavior when you call `saveHTML()` multiple times, it could be that this has resolved it.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/4019/commits/1e629af85a318b1e6511170689e26d7ac0b56d00